styles/index.scss

```diff
@@ -0,0 +1,2 @@
+@import 'styles/uswds-dependencies.scss';
```

Comment:
for consistency's sake let's import the ReactUSWDS css here also (instead of in `_app.tsx`)